.doc_gen/cross-content/cross_RekognitionPhotoAnalyzer_Python_block.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd"[
+    <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+    %phrases-shared;
+]>
+<block>
+    <para>
+    Shows you how to use the &Python3long; to create a web application that
+    lets you upload photos to an &S3long; (&S3;) bucket, use &REKlong;
+    to analyze and label the photos, and use &SESlong; (&SES;) to send email
+    reports of image analysis.
+    </para>
```

Comment:
`<para>Shows you how to use the &Python3long; to create a web application that
    lets you do the following:</para>
<itemizedlist><listitem><para>Upload photos to an &S3long; (&S3;) bucket.</para></listitem>
        <listitem><para>Use &REKlong;
    to analyze and label the photos.</para></listitem>
        <listitem><para>Use &SESlong; (&SES;) to send email
    reports of image analysis.</para></listitem>
</itemizedlist>`

Reply:
Done